package.json

```diff
@@ -199,10 +199,12 @@
       "v6",
       {
         "name": "v6-beta",
+        "channel": "beta",
         "prerelease": "beta"
       },
       {
         "name": "v7",
+        "channel": "alpha",
         "prerelease": "alpha"
```

Comment:
Same comment as above

Reply:
Same answer as above. It is needed to make releases like `7.0.0-alpha` instead of `7.0.0` from the v7 branch. Once v7 is a stable release channel, it should be a bare "v7" like the "v6" branch reference (or just remove the "channel" and "prerelease" properties, leaving only "name").